src/components/Yritys/ExistingYritys.svelte

```diff
@@ -36,6 +37,10 @@
     },
     { text: $_('yritys.laatijat') }
   ];
+
+  $: Breadcrumb.set([[$_('yritys.yritykset'), '/#/yritykset'],
+                     [Maybe.fold('...', R.prop('nimi'), yritys), window.location.href]]);
```

Comment:
Could be just me, but i think an object would be better fit here instead of array posing as tuple.

Reply:
Changed.